packages/http-front-cache/src/lib/remove-cache-by-param-session-storage.ts

```diff
@@ -0,0 +1,12 @@
+import { sessionStorageProvider } from './providers/session-storage'
+import { removeCacheByParam } from './remove-cache-by-param'
+
+/**
+ * Delete the cache on sessionStorage by param (where param is the same param provided to the serviceFunction)
+ * @param params: the parameters that will be used find the cache key (hash)
+ */
+export const removeCacheByParamOnSessionStorage = (
```

Comment:
It is specific for sessionStorage, as the http-front-cache already provides the cacheOnSessionStorage